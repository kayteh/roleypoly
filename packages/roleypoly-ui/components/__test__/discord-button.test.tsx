/**
 *  @jest-environment jsdom
 */
/* eslint-env jest */
import * as React from 'react'
// import renderer from 'react-test-renderer'
import { shallow } from 'enzyme'
import DiscordButton from '../discord-button'
import 'jest-styled-components'

describe('<DiscordButton />', () => {
  it('renders correctly', () => {
    const button = shallow(<DiscordButton>Hello!</DiscordButton>)
    expect(button).toMatchSnapshot()
    expect(button.text().trim()).toEqual('Hello!')
  })
})
