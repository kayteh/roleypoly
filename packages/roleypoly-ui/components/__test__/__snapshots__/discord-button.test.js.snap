// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DiscordButton /> renders correctly 1`] = `
<discord-button__Button>
  <discord-button__ButtonIcon
    src="/static/discord-logo.svg"
  />
  Â 
  Hello!
</discord-button__Button>
`;
